packages/tf-inventory/src/inventory.js

```diff
@@ -81,7 +81,7 @@ async function listAwsResources() {
   ec2Instances = [].concat(...ec2Instances.Reservations.map((e) => e.Instances));
   ec2Instances = ec2Instances.map((inst) => inst.InstanceId);
 
-  let esDomainNames = await es().listDomainNames().promise();
+  let esDomainNames = await es().listDomainNames();
```

Comment:
Looks totally right to me. This seems like the most obvious spot for an issue but 🤷